round4/fehra.md

```diff
@@ -0,0 +1,108 @@
+
+## Selling Points
+
+1. Pop-up notifications prevent having to spend time switching between product page and budget spreadsheet.
+2. Works across many big online retailers such as Amazon, Ebay, etc.
+3. Considers an item automatically into your budget when that item is purchased.
+4. Synchronizes with traditional budgeting tools like Google Sheets.
+5. Easy to adjust budgets, and provides recommended budget breakdowns.
+
+## User Stories
+
+- As someone who is on a budget, I want easily track if the things I want to buy online go over my budget so that I don’t overspend
+
+- As someone who likes to track my purchases, I want to track the category of items I am buying and the total money spent in each category so that I can track the category of items I’m overspending on.
+
+- As an avid budgeter, I want to see the percentage of my budget that I have left, so that I can visualize how much of my budget I have remaining
+
+- As someone who shops on multiple websites, I want to track the total cost of everything in my cart on all of the websites, so that I can see how much everything costs across all the websites.
+
+- As someone who tends to make splurge purchases, I want to view prices as percentages of my budget so that I’m more aware of the consequences of my habits
+
+- As someone who maintains budgeting spreadsheets, I want to update my sheets remotely upon purchase so that I never forget to log anything
+
+- As someone trying to learn the value of money, I want to be able to visualize prices as hours worked so that I’m conscious about how much a dollar costs me
+
+- As a regular concertgoer, I want to be able to factor in ticket prices well in advance so that I know I’ll have enough by the time tickets are released
+
+- As a forgetful person, I want to know when a purchase uses up a substantial amount of my budget so that I never forget about my monthly limits
+
+- As someone who regularly orders international items, I want to be able to convert foreign currencies effortlessly so that I can correctly factor them into my budget
+
+- As a frequent traveler, I want to be able to allocate travel funds ahead of time so that I can regulate my other expenses accordingly
+
+- As a freelancer with irregular income, I want to be able to adjust my budget easily so that I can spend according to my current income
+
+- As someone with dependents, I want to be able to set aside money for them so that I never fail to give them their money on time
+
+- As someone new to budgeting, I want to receive a recommended breakdown of how to divide my funds so that I know how to start off
+
+- As a taxpayer, I want to have my budget automatically factor in my tax amount so that I’ll have enough saved come tax season.
+
+
+## Notes
+
+Might have to limit ourselves to sites where prices can easily be scraped. 
```

Comment:
I like that you are thinking about how you are going to have to limit yourself. I recommend you guys think about how you are going to scrape the sites because usually sites are pretty hard to scrape at a surface level without hardcoding a lot of different words to look for. Most sites are usually locked down and it will be interesting how you will create a backend to be able to look for key words to suggest the budget since you cannot hardcode all the sites that one may use